src/main/webapp/style.css

```diff
@@ -0,0 +1,7 @@
+h1 {
+  font-size: x-large;
+}
+
+.emphasis {
+  color: violet;
```

Comment:
Also, if you think that we'll be moving forward with violet as our accent color, it might be worth looking into [theming Boostrap](https://getbootstrap.com/docs/4.0/getting-started/theming/#modify-map) to change their built-in color scheme to match what we're looking for.

Reply:
Although I've never worked with Sass before so maybe leave this as a nice-to-have for now. Don't need to overcomplicate things.

Reply:
yeah I think making a class with CSS should be fine.

Reply:
It get's the job done and you can always call it if the need arises

Reply:
I agree. Just throwing out options b/c Bootstrap is cool

Reply:
@Richie78321 what is Sass? Also I will def look into Bootstrap for the next iteration if that will make our site more consistent. If you want to go with general theming (which looks really cool) maybe we can decide on the specifics during Pod Sunk?